datasets/flwr_datasets/federated_dataset.py

```diff
@@ -213,6 +214,72 @@ def load_full(self, split: str) -> Dataset:
         self._check_if_split_present(split)
         return self._dataset[split]
 
+    def concatenate_divisions(
+        self, division_id: Union[int, str], split: Optional[str] = None
+    ) -> Dataset:
+        """Concatenate the divisions of the partitions.
+
+        The divisions are created based on the `partition_division` and accessed based
+        on the `division_id`. If you specified the divisions for more than 1 partitioner
+        you need to specify the split associated with a partitioner that creates the
+        divisions. It can be used to create e.g. centralized dataset from federated
+        on-edge test sets.
+        sets
+
+        Parameters
+        ----------
+        division_id : Union[int, str]
+            The way to access the division (from a List or DatasetDict).
```

Comment:
This is not very clear. If i look at the tests associated with this method, to me it seems `division_id` is the same as the `split` (str) used for `partitioners`? We are introducing a new `division_id` term just for this method?